docs/gke.md

```diff
@@ -0,0 +1,73 @@
+# Google Kubernetes Engine (GKE) Deployment Guide
+
+Federation v2 can be deployed to and manage [GKE](https://cloud.google.com/kubernetes-engine/) clusters. Since
+Federation v2 requires Kubernetes v1.11 or greater, you must create
+[Alpha Clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/alpha-clusters) and manually specify
+the `cluster-version`. Alpha clusters will no longer be needed once GKE supports Kubernetes `v1.11` or greater. You can
+use the `$ gcloud container get-server-config` command to view the GKE cluster versions available to you.
+The following example deploys two GKE alpha clusters named `cluster1` and `cluster2` using Kubernetes version
+`1.11.2-gke.4`.
+
+```bash
+$ gcloud container clusters create cluster1 --enable-kubernetes-alpha --cluster-version 1.11.2-gke.4 \
+--no-enable-autorepair --no-enable-autoupgrade
+$ gcloud container clusters create cluster2 --enable-kubernetes-alpha --cluster-version 1.11.2-gke.4 \
+--no-enable-autorepair --no-enable-autoupgrade
+```
+**NOTE:** GKE alpha clusters expire after 30 days.
+
+Configure cluster access for `kubectl`:
+```bash
+$ gcloud container clusters get-credentials cluster1
```

Comment:
(no action required) This adds the necessary credentials to your kubeconfig and is normally done automatically for you once the `gcloud container clusters create` command returns.

Reply:
OK, I'll remove this step.